samples/snippets/src/test/java/webrisk/SubmitUriExampleTest.java

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+
+package webrisk;
+
+import com.google.common.truth.Truth;
+import com.google.webrisk.v1.Submission;
+import org.junit.Test;
+
+public class SubmitUriExampleTest {
+
+  @Test
+  public void testSumbitUriExample() {
+    Submission actualSubmission = SubmitUriExample.submitUriExample();
+    Truth.assertThat(actualSubmission.getUri()).isEqualTo("http://testsafebrowsing.appspot.com/s/malware.html");
```

Comment:
For testing, let's pass the value to submitUriExample with a constant or a variable and assert the same value in the response.